# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/WaylandBuffer.h
# Commit: 8af28b4f8ccb
# Full Hash: 8af28b4f8ccbdf4c317d3f382d430d2df7868949
# Author: stransky <stransky@redhat.com>
# Date: 2024-12-17 17:00:49
# Regressor Bug: 1934497
# File Overlap Count: 2
# Description:
#   Bug 1934497 [Wayland] Fix non unified build bustage r=emilio CLOSED TREE
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D232322
# ==============================================================================

diff -r 58a807c8fde7 -r 8af28b4f8ccb widget/gtk/WaylandBuffer.h
--- a/widget/gtk/WaylandBuffer.h	Tue Dec 17 08:03:59 2024 +0000
+++ b/widget/gtk/WaylandBuffer.h	Tue Dec 17 08:53:58 2024 +0000
@@ -17,11 +17,10 @@
 #include "mozilla/RefPtr.h"
 #include "nsTArray.h"
 #include "nsWaylandDisplay.h"
+#include "WaylandSurface.h"
 
 namespace mozilla::widget {
 
-class WaylandSurface;
-
 // Allocates and owns shared memory for Wayland drawing surface
 class WaylandShmPool {
  public: